cypress/pageObjects/products_page_object.js

```diff
@@ -0,0 +1,38 @@
+const visitPage = () => {
+    cy.visit('http://localhost:4200/')
+}
+
+const addProduct = (buttonId) => {
```

Comment:
this is doing the same thing as `cleanTheBasket`, I mean code is the same.
If you were to re name it to clickById(), then you could reuse it elsewhere without duplication.
clickById(Elements.productOne)
OR
something what can be more reusable and have similar semantics.
clickById(Elements.addToBasketOne)